src/Core/src/Core.csproj

```diff
@@ -24,6 +24,7 @@
   </ItemGroup>
   <ItemGroup Condition="$(TargetFramework.Contains('-windows'))">
     <PackageReference Include="Microsoft.WindowsAppSDK" />
+    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" />
```

Comment:
We can remove this line once WASDK updates to depend on the newer version - or we may just keep this so we control the base version.